packages/custom-token-bridging/scripts/exec-viaLib.js

```diff
@@ -0,0 +1,108 @@
+const { providers, Wallet } = require('ethers')
+const { getL2Network, L1ToL2MessageStatus } = require("arb-ts")
+const { arbLog, requireEnvVariables } = require('arb-shared-dependencies')
+const { AdminErc20Bridger } = require('arb-ts/dist/lib/assetBridger/erc20Bridger')
```

Comment:
```suggestion
const { AdminErc20Bridger } = require('arb-ts')
```

Reply:
Hmm, also looks like this isnt been exported, will add this to the index